<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPhotoList</class>
 <widget class="QWidget" name="FormPhotoList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ori_uis.qrc">
    <normaloff>:/GUIS/photo.png</normaloff>:/GUIS/photo.png</iconset>
  </property>
  <widget class="QLabel" name="labelThumbnail">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>270</y>
     <width>441</width>
     <height>361</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>571</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(100, 100, 100);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReturn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonThumbnail">
   <property name="geometry">
    <rect>
     <x>1630</x>
     <y>80</y>
     <width>94</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Thumbnail</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBluetooth">
   <property name="geometry">
    <rect>
     <x>1800</x>
     <y>1010</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAirplay">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>1010</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBottom">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>980</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>harman/kardon</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWifi">
   <property name="geometry">
    <rect>
     <x>1730</x>
     <y>1010</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResolution">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>770</y>
     <width>1901</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Resolution: 1920X1080</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPhotoText">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>119</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Photos</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPhoto">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>125</width>
     <height>125</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/bds_UIs/image120.PNG</pixmap>
   </property>
  </widget>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>horizontalLayoutWidget_2</zorder>
  <zorder>horizontalLayoutWidget_3</zorder>
  <zorder>horizontalLayoutWidget_4</zorder>
  <zorder>labelBluetooth</zorder>
  <zorder>labelResolution</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
  <include location="ori_uis.qrc"/>
 </resources>
 <connections/>
</ui>
